src/commands/build.yml

```diff
@@ -72,7 +83,13 @@ steps:
   - run:
       name: <<parameters.step-name>>
       command: |
+        IFS="," read -ra CACHE_FROM_IMAGES \<<< "<< parameters.cache_from >>"
+        for image in "${CACHE_FROM_IMAGES[@]}"; do
+          docker pull $image || echo "Failed to pull the specified image, $image. Ignoring..."
+        done
+
         docker build \
+          --cache-from "<<parameters.cache_from>>" \
```

Comment:
Looking at #27 doesn't this require a condition and unless version as well?

Reply:
The quotes around it take care of that. Docker allows an empty —cache-from which means just go to disk and use all the cache layers available. Since the default value of the param is empty string, it’s equal to `—cache-from “”`.